board/opendingux/package/mininit/mininit.mk

```diff
@@ -0,0 +1,18 @@
+#############################################################
+#
+# mininit
+#
+#############################################################
+
+MININIT_VERSION = 429a1d019a9a1fe5d3662567cd958a54d963e0ff
```

Comment:
Could you short it?
```git rev-parse --short 429a1d019a9a1fe5d3662567cd958a54d963e0ff``` in the mininit repo.

Reply:
Done